CHANGES.txt

```diff
@@ -1,3 +1,6 @@
+1.4.2
+ - Added `sync.enabled` property to SDK configuration to allow synchronize splits and MySegments once
```

Comment:
I would use `segments only once.` instead of `MySegments once`, because it also apply for server-side segments